CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## 2.48.0-rc.0 / 2023-10-10
+
+* [CHANGE] Remote-write: respect Retry-After header on 5xx errors. #12677
```

Comment:
Wasn't sure about calling this a change or an enhancement, but it does change the current behavior of remote-write retries

Reply:
Could see it as a bugfix, even.

Reply:
Yeah, I just don't know how expected that behavior is. Especially as its not a configuration option. But, I'm up to whatever you want.

Also on another note I see in that PR there was some discussion around documenting it, but I'm not sure if that that ever happened?

Reply:
it's a change to me